src/client/app/components/ChartSelectComponent.tsx

```diff
@@ -40,32 +50,37 @@ export default class ChartSelectComponent extends React.Component<ChartSelectPro
 				<p style={labelStyle}>
 					<FormattedMessage id='graph.type' />:
 				</p>
-				<ButtonGroup>
-					<Button
-						outline={this.props.selectedChart !== ChartTypes.line}
-						onClick={() => this.handleChangeChartType(ChartTypes.line)}
-					>
-						<FormattedMessage id='line' />
-					</Button>
-					<Button
-						outline={this.props.selectedChart !== ChartTypes.bar}
-						onClick={() => this.handleChangeChartType(ChartTypes.bar)}
-					>
-						<FormattedMessage id='bar' />
-					</Button>
-					<Button
-						outline={this.props.selectedChart !== ChartTypes.compare}
-						onClick={() => this.handleChangeChartType(ChartTypes.compare)}
-					>
-						<FormattedMessage id='compare' />
-					</Button>
-					<Button
-						outline={this.props.selectedChart !== ChartTypes.map}
-						onClick={() => this.handleChangeChartType(ChartTypes.map)}
-					>
-						<FormattedMessage id='map' />
-					</Button>
-				</ButtonGroup>
+				<Dropdown isOpen={this.state.dropdownOpen} toggle={this.toggleDropdown}>
+					<DropdownToggle caret>
+						<FormattedMessage id='Select a graph type' />
```

Comment:
The "Select a graph type" is not language aware so it does not change with selected language.

Reply:
I have updated the pull request to reflect the FormattedMessage so it is language aware. 

Also, I've kept the style the same for now, but I've posted screenshots to show how the dropdown may look with the proposed ideas. Take a look at the lower conversation for the link.